doc/apiary/v2/fiware-ngsi-simple.apib

```diff
@@ -50,16 +50,136 @@ to keep your client decoupled from implementation details.
 
 ## Context Entities [/v2/entities{?type,pattern,limit,offset}]
 
-### List entities [GET /v2/entities{?limit,offset,count,type,id}]
-
-Detailed description
-
-+ Parameters
-    + limit: 20 (optional, number) - Limit the number of entities to be retrieved
-    + offset: 20 (optional, number) - Skip a number of records. Pagination
-    + type: Room (optional, string) - Only retrieve entities of this type
-    + id: Bcn-Welt (optional, string) - Retrieve entities by id
-    + idPattern: (optional, string) - Retrieve entities by pattern id
+### List entities [GET /v2/entities{?limit,offset,count,type,id,idPattern,q,geometry,coords,attrs}]
+
+Retrieves a list of entities, according to a set of matching criteria defined by the `id`, `type`, `idPattern`, `q`,
+`geometry` and `coords` attribute (see below for a detailed description of these parameters). A given entity
```

Comment:
Retrieves a list of entities which match criteria defined by the following parameters: 

<And here we can put the description table>